src/test/test-malloc.c

```diff
@@ -0,0 +1,179 @@
+/*======================================================================
+FILE: regression-malloc.c
+
+(C) COPYRIGHT 2018, Markus Minichmayr
+
+This library is free software; you can redistribute it and/or modify
+it under the terms of either:
+
+The LGPL as published by the Free Software Foundation, version
+2.1, available at: http://www.gnu.org/licenses/lgpl-2.1.html
+
+Or:
+
+The Mozilla Public License Version 2.0. You may obtain a copy of
+the License at http://www.mozilla.org/MPL/
+======================================================================*/
+
+#ifdef HAVE_CONFIG_H
+#include <config.h>
+#endif
+
+#include "libical/ical.h"
+
+#include "regression.h"
+#include "test-malloc.h"

```

Comment:
I like to include the functional unit's header above the others.
like so:
```
#include "test-malloc.h"
#include "regression.h"
```
I don't think `#include libical/ical.h` is necessary.